csharp/src/Ice/IceDiscovery/Locator.cs

```diff
@@ -85,7 +94,7 @@ public ValueTask SetAdapterDirectProxyAsync(string adapterId, IObjectPrx? proxy,
                     }
                 }
             }
-            return new ValueTask();
+            return default;
```

Comment:
The problem with `default` and `new ValueTask()` the preferred form was `new ValueTask(Task.CompletedTask)` this was because the behavior of the others was not documented.

I opened an issue with .NET to get it documented https://github.com/dotnet/dotnet-api-docs/issues/4397
I think it is fine to use it now that it is documented

Reply:
Let's use default then!